### DISCUSSION TITLE
What are the units of measure for the flow rate?

### DISCUSSION THREAD

**Discussion by mtbsteve** [jomjol/AI-on-the-edge-device#2030] [Q&A] [answered]
In iobroker, I am getting the following rate values (snapshot) for:
- rate_per_digitization_round: 0,00186
- rate_per_time_unit: 0,00062

So I assume that given my defined scan interval of 3 minutes, my time_unit is in Minutes, so my current flow rate is 0,00062 of "volume" per minute. It looks to me it is cubic meters per minute. Is my assumption correct?
Is there a way to change it eg to liters per minute?
The problem is that when I store the values in influxdb, some rounding issues appear. In the example above, influx stores only 4 decimal places, and 0,00062 is stored as 0,0006

Edit: the parameter "decimal_shift" fixes the rate unit of measure, but when I set it to "3", the unit of measure for the main water counter value also switched to "liters" which is a bit cumbersome.

**Comment by jomjol** [comment, marked as answer]
Yes you are right, the value is m³/minute. Background is, that this is also for power and gas meters and therefore it is hard to say, what is the right base.
But you can do a trick: you can shift the decimater (",") by 3 digits to the right: 3.345 m³ --> 3345 Liter

**Reply by mtbsteve** [reply]
Thanks - everything works as expected now. Fantastic job!
